### PR TITLE
DDBEASY-109 - Various fixes.

### PR DESCRIPTION
**Pull request by guddo**

### CODE REVIEW

File: ting_material_details.field.inc [hunk 1]
```diff
@@ -413,12 +413,23 @@ function ting_material_details_field_formatter_view($entity_type, $entity, $fiel
...
         case 'ting_details_single':
           $details = '';
+
           if (is_array($val)) {
             foreach ($val as $detail) {
               $details .= '<span class="ting-material-detail">' . $detail . '</span>';
             }
           }
           else {
+            if ($field['field_name'] == 'ting_details_contributor') {
```
Comment (xendk) [inline ting_material_details.field.inc]: Checking field_name in the formatter is not the way. Create a new formatter.

Comment (guddo) [inline ting_material_details.field.inc]: New commit added.
/cc @xendk

File: ting_material_details.field.inc [hunk 2]
```diff
@@ -426,13 +433,25 @@ function ting_material_details_field_formatter_view($entity_type, $entity, $fiel
...
+          break;
+
         case 'ting_details_creator_search':
           if (!is_array($val)) {
             $val = array($val);
           }
           $links = array();
           foreach ($val as $value) {
-            $links[] = l($value, 'search/ting/phrase.creator=' . $value);
+            $links[] = l($value, 'search/ting/phrase.creator="' . $value .'"');
```
Comment (xendk) [inline ting_material_details.field.inc]: How about building on #7, depending on the core teams response to it?

**Comment by cableman** [comment]
We are closing the ding2tal project as the process of moving the code into DDB CMS / ding2 core is almost completed.

So please re-do this pull request against the ding2 core repositories found at http://github.com/ding2, so the core team can do the review. Remember to create a ticket/issue at http://platform.dandigbib.org/projects/ddb-cms before making the pull request to ding2.

If you have any questions, please send a mail to dingcore@ting.dk.

**Comment by guddo** [comment]
Moved to https://github.com/ding2/ting_material_details/pull/7.

**Pull request closed without merging**